docs/extensibility/README.md

```diff
@@ -2,7 +2,9 @@
 
 ## Overview
 
-Extensibility in Busola is a feature that allows you to create a separate dedicated user interface (UI) page for your CustomResourceDefinition (CRD). It allows you to add navigation nodes, on Cluster or Namespace level, and to configure your resource list page, details pages, as well as create and edit forms. You can also add [display](display-section.md) and [form](form-section.md) widgets to have a graphical representation of the elements.
+Extensibility in Busola is a feature that allows you to create a separate dedicated user interface (UI) page for your CustomResourceDefinition (CRD). It allows you to add navigation nodes, on Cluster or Namespace level, and to configure your [UI display](display-section.md), for example, a resource list page, and details pages. You can also [create and edit forms](form-section.md).
```

Comment:
```suggestion
With Busola's extensibility feature, you can create a separate dedicated user interface (UI) page for your CustomResourceDefinition (CRD). It enables you to add navigation nodes, on cluster or Namespace level, and to configure your [UI display](display-section.md), for example, a resource list page, and details pages. You can also [create and edit forms](form-section.md).
```

Is "cluster" capitalized? 
Otherwise, just my usual allergy against "allows" when we're talking about stuff that never was forbidden.